Release 0.6.0 (#37)
nginxinc/nginx-plus-go-client: CHANGELOG.md

## 0.6.0 (Nov 8, 2019)
FEATURES:
* [34](https://github.com/nginxinc/nginx-plus-go-client/pull/34): *Support for updating upstream servers parameters*. The client now supports to update upstream parameters of servers that already exist in NGINX Plus.

CHANGES:

> * Public methods `UpdateHTTPServers` and `UpdateStreamServers` now return a third slice of servers if the updated servers are already present in NGINX Plus but they have different parameters. 

Reviewer: ```suggestion
* Public methods `UpdateHTTPServers` and `UpdateStreamServers` now return a third slice that includes the updated servers -- i.e. the servers that were already present in NGINX Plus but were updated with different parameters. 
```
